docs/wip/metadata-tests.md

```diff
@@ -1,7 +1,281 @@
+will be applied continuously over time, as the selection set evolves & changes with your data ecosystem.
+
+When defining selection criteria, you'll be able to choose from a range of useful technical signals (e.g. usage, size) that are automatically
+extracted by DataHub (which vary by integration). This makes automatically classifying the "important" assets in your organization quite easy, which
+is in turn critical for running effective Data Governance initiatives within your organization. 
+
+For example, we can define a Metadata Test which selects all Snowflake Tables which are in the top 10% of "most queried"
+for the past 30 days, and then assign those Tables to a special "Tier 1" group using DataHub Tags, Glossary Terms, or Domains. 
+
+
+### Automated Data Governance Monitoring
+
+Metadata Tests allow you to define & monitor a set of rules that apply to assets in your data ecosystem (e.g. datasets, dashboards, etc). This is particularly useful when attempting to govern 
+your data, as it allows for the (1) definition and (2) measurement of centralized metadata standards, which are key for both bootstrapping
+and maintaining a well-governed data ecosystem. 
+
+For example, we can define a Metadata Test which requires that all "Tier 1" data assets (e.g. those marked with a special Tag or Glossary Term),
+must have the following metadata:
+
+1. At least 1 explicit owner *and*
+2. High-level, human-authored documentation *and*
+3. At least 1 Glossary Term from the "Classification" Term Group
+
+Then, we can closely monitor which assets are passing and failing these rules as we work to improve things over time.
+We can easily identify assets that are *in* and *out of* compliance with a set of centrally-defined standards. 
+
+By applying automation, Metadata Tests 
+can enable the full lifecycle of complex Data Governance initiatives - from scoping to execution to monitoring. 
+
+
+## Metadata Tests Setup, Prerequisites, and Permissions
+
+What you need to manage Metadata Tests on DataHub:
+
+* **Manage Tests** Privilege
+
+This Platform Privilege allows users to create, edit, and remove all Metadata Tests on DataHub. Therefore, it should only be
+given to those users who will be serving as metadata Admins of the platform. The default `Admin` role has this Privilege.
+
+> Note that the Metadata Tests feature is currently limited in support for the following DataHub Asset Types:
+>
+>- Dataset
+>- Dashboard
+>- Chart
+>- Data Flow (e.g. Pipeline)
+>- Data Job (e.g. Task)
+>- Container (Database, Schema, Project)
+> 
+> If you'd like to see Metadata Tests for other asset types, please let your Acryl Customer Success partner know!
+
+## Using Metadata Tests
+
+Metadata Tests can be created by first navigating to **Govern > Tests**.
+
+To begin building a new Metadata, click **Create new Test**.
+
+<p align="center">
+  <img width="40%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/manage-tests.png"/>
```

Comment:
The image is super tough to read at this size - let's bump it up to 75%

Reply:
Thanks!